ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -45,3 +47,32 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+
+
+#tensordot v1
+@given(
+    dtype_and_inputs_axes=test_linalg._get_dtype_value1_value2_axis_for_tensordot(ivy_tf.valid_float_dtypes),
+    as_variable=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.tensordot"
+    ),
+    native_array=st.booleans(),
+)
+def test_tensorflow_tensordot(
+    dtype_and_inputs_axes, as_variable, num_positional_args, native_array, fw
+):
+    dtype, x, y, axes= dtype_and_inputs_axes
```

Comment:
there should be single space both to the left and right of `=`

Reply:
Thank you for your help. I'll change that